src/modules/esl-share/core/esl-share-trigger.ts

```diff
@@ -1,37 +1,103 @@
+   * the prefix `group:`) separated by spaces.
+   * @example "facebook reddit group:default"
+   * */
+  @attr({defaultValue: 'all'}) public list: string;
+  /** Popup state marker */
+  @boolAttr() public popupShown: boolean;
+  /** Hover event tracking media query. Default: `all` */
+  @attr({defaultValue: 'all'}) public override trackHover: string;
+
+  /** Default initial params to pass into the popup */
+  @jsonAttr<ESLSharePopupActionParams>({defaultValue: {
+    position: 'top',
+    hideDelay: 220
+  }}) public popupInitialParams: ESLSharePopupActionParams;
+
+  /** Target observable Toggleable */
+  public override get $target(): ESLToggleable | null {
+    return ESLSharePopup.sharedInstance;
+  }
+  public override set $target(value: any) {}
+
+  /** The text writing directionality of the element */
+  protected get currentDir(): string {
+    return getComputedStyle(this).direction;
+  }
+
+  /** The base language of the element */
+  protected get currentLang(): string {
+    const el = this.closest('[lang]');
+    return (el) ? (el as HTMLElement).lang : '';
+  }
+
+  /** Container element that defines bounds of popups visibility */
+  protected get containerEl(): HTMLElement | undefined {
+    const container = this.getClosestRelatedAttr('container');
+    return container ? ESLTraversingQuery.first(container, this) as HTMLElement : undefined;
+  }
 
-  /** Shows the target {@link ESLPopup} with a passed params */
+  /** Update `$target` Toggleable  from `target` selector */
+  public override updateTargetFromSelector(): void {}
+
+  /** Gets attribute value from the closest element with group behavior settings */
+  protected getClosestRelatedAttr(attrName: string): string | null {
+    const relatedAttrName = `${this.baseTagName}-${attrName}`;
+    const $closest = this.closest(`[${relatedAttrName}]`);
+    return $closest ? $closest.getAttribute(relatedAttrName) : null;
+  }
+
+  /** Merges params to pass to the toggleable */
+  protected override mergeToggleableParams(this: ESLShareTrigger, ...params: ESLSharePopupActionParams[]): ESLSharePopupActionParams {
+    const {list, containerEl, currentDir: dir, currentLang: lang} = this;
+    return Object.assign({
+      initiator: 'share',
+      activator: this,
+      containerEl,
+      list,
+      dir,
+      lang
+    }, this.popupInitialParams, ...params);
+  }
+
+  /** Shows popup with passed params */
   public override showTarget(params: ESLToggleableActionParams = {}): void {
-    super.showTarget(params);
+    const actionParams = this.mergeToggleableParams({}, params);
+    if (ESLSharePopup.open) this.hideTarget();
+    ESLSharePopup.show(actionParams);
+  }
 
-    this.forwardAttributes();
+  /** Hides popup with passed params */
+  public override hideTarget(params: ESLToggleableActionParams = {}): void {
+    const actionParams = this.mergeToggleableParams({}, params);
+    ESLSharePopup.hide(actionParams);
   }
+}
```

Comment:
Do we still need to override it, even when the `$target` is declared constantly to ESLSharePopup ?

Reply:
I guess they are not needed after resolving the issue #1677